.github/workflows/php.yml

```diff
@@ -14,18 +14,21 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php-versions: ['5.6', '7.0', '7.1', '7.2', '7.3', '7.4']
+        php-versions: ['7.2', '7.3', '7.4']
 
     steps:
     - uses: actions/checkout@v2
 
+    - uses: nanasess/setup-php@master
```

Comment:
Did you consider https://github.com/shivammathur/setup-php instead of this?

Reply:
didn't even know it existed 😆 i just got this one from Googling so happy to swap if you have a preference.

Reply:
Yeah, I'd recommend try it out. That user has many more stars and other PHP-related projects.

Reply:
sure, swapped in 0465c9c